apps/daffio/src/app/docs/components/doc-viewer/doc-viewer.component.html

```diff
@@ -3,10 +3,18 @@
 	<div>Menu</div>
 </button>
 <div class="daffio-doc-viewer__grid">
-	<daff-article
-		class="daffio-doc-viewer__content"
-		[innerHtml]="sanitizedContent">
-	</daff-article>
+  @if (isApiPackage) {
+    <daff-article class="daffio-doc-viewer__content">
+      <h1>{{doc.title}}</h1>
+      <h2>Exports</h2>
```

Comment:
Why do we need to label it as exports?

Reply:
The page looked really empty without it honestly but you can do whatever design seems best to you; I don't have any strong preference.

Reply:
Can we add a sidebar that lists out all the packages, similar to the individual `docs/packages` pages (w/ links to the package api page)

Reply:
yeah I think that should be a different PR, can you add an issue for it?